Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -7010,6 +7015,113 @@ local Item5 = cItem(E_ITEM_DIAMOND_CHESTPLATE, 1, 0, "thorns=1;unbreaking=3");
 				},
 			},
 		},
+		cBookContent =
+		{
+			Desc = [[
+This class contains the information for a signed or writeable book: The author, title and the pages. A page of the writeable book is a simple string. For a signed book it can be a json string. For the json string use {{cCompositeChat}} to create a page and then the function {{cCompositeChat#CreateJsonString_1|CreateJsonString}} to get the json string.
+]],
+			Functions =
+			{
+				constructor =
+				{
+					Returns =
+					{
+						{
+							Type = "cBookContent",
+						},
+					},
+					Notes = "Creates a empty book",
+				},
+				SetAuthor =
+				{
+					Params =
+					{
+						{
+							Name = "Author",
+							Type = "string",
+						},
+					},
+					Notes = "Set the author of the book",
+				},
+				GetAuthor =
+				{
+					Returns =
+					{
+						{
+							Type = "string",
+						},
+					},
+					Notes = "Get the author of the book",
+				},
+				SetTitle =
+				{
+					Params =
+					{
+						{
+							Name = "Title",
+							Type = "string",
+						},
+					},
+					Notes = "Set the title of the book",
+				},
+				GetTitle =
+				{
+					Returns =
+					{
+						{
+							Type = "string",
+						},
+					},
+					Notes = "Returns the title of the book",
+				},
+				AddPage =
+				{
+					Params =
+					{
+						{
+							Name = "Page",
+							Type = "string",
+						},
+					},
+					Notes = "Add a page to the end of the book. Note: If it's a written book, the page can be a json string",
+				},
+				GetPages =
+				{
+					Returns =
+					{
+						{
+							Type = "table",
+						},
+					},
+					Notes = "Returns the pages of the book as a table",
+				},
+				SetPages =
+				{
+					Params =
+					{
+						{
+							Name = "Pages",
+							Type = "table",
+						},
+					},
+					Notes = "Set the pages of the book",
```

Comment:
`Sets all the pages of the book`

How about adding a `SetPage(PageNum, Text)`, and appropriate `GetPage(PageNum)`